CHANGES_NEXT_RELEASE

```diff
@@ -19,6 +19,7 @@
 - Fix: Allowing trailing semicolon in a string filter, but only one (Issue #2086)
 - Fix: NGSIv2 geoquery syntax errors using 400 Bad Request instead of 422 Unprocessable Entity (according to rules defined in #1286)
 - Fix: NGSIv2 URI PATH must be all in lowercase, otherwise "service not found" (Issue #2057)
+- Add: New URI param 'mq' for matching in metadata (Issue #1156) 
```

Comment:
Actually, what you have achieve in this PR is much larger :), covering not only "mq" in URI param for GET but also FIWARE::StringQuery::Metadata.

Thus, I'd suggest the following 2 lines:

- Add: New param 'mq' (both in URI param for list all entities operation and expression sub-field in subscriptions) for matching in metadata (#1556)
- Add: FIWARE::StringQuery::Metadata scope (#1556)